org.apache.airavata.django.tests/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -0,0 +1,11 @@
+eclipse.preferences.version=1
```

Comment:
Remove this file. This is specific to local setup. All auto generated files should be removed